LibCarla/source/carla/MoveHandler.h

```diff
@@ -19,8 +19,8 @@ namespace detail {
     MoveWrapper(MoveWrapper &&) = default;
     MoveWrapper& operator=(MoveWrapper &&) = default;
 
-    MoveWrapper(const MoveWrapper &);
-    MoveWrapper& operator=(const MoveWrapper &);
+    MoveWrapper(const MoveWrapper &) = delete;
+    MoveWrapper& operator=(const MoveWrapper &) = delete;
```

Comment:
It is a hack to trick asio into accepting move-only handlers from [https://stackoverflow.com/a/22891509](https://stackoverflow.com/a/22891509).

Reply:
Thanks to this hack we save one dynamic allocation per RPC message

Reply:
Exactly! It is wonderful. The server-client code is perfect. I cannot optimize any piece of the code.